src/pyasm/search/sql_dumper.py

```diff
@@ -178,6 +180,15 @@ def set_include_id(my, flag=True):
 
     def set_ignore_columns(my, columns=[]):
         my.ignore_columns = columns
+    
+    
+    def set_replace_token(my, project_code, replace, regex, column):
```

Comment:
regex needs to be put as the last arg.. and as an optional kwarg like
regex = None